MAINTAINING.md

```diff
@@ -33,6 +36,14 @@
     * Documents in the `docs` directory are consistent with new changes
     * All combinators have time and space complexity annotations
     * All combinators have `since` annotation
+    * Unreleased combinators should be marked with `pre-release` or
+      `internal` annotations
+
+* _Build and Test_:
+
+    * All CIs are green
+    * Manually build and test for all flags (esp. `dev` flag) not covered by CIs
```

Comment:
We should probably list the flags here (or somewhere else) for convinence.

Reply:
They are all listed on the hackage index page and in the cabal file.